src/resources/views/base/auth/login.blade.php

```diff
@@ -57,7 +57,7 @@
                     </form>
                 </div>
             </div>
-            @if (backpack_users_have_email() && config('backpack.base.setup_password_recovery_routes', true))
+            @if (backpack_users_have_email() && backpack_email_column() == 'email' && config('backpack.base.setup_password_recovery_routes', true))
```

Comment:
Hey @jorgetwgroup So if you set `email_column` to be `mail` instead of `email`, this would not work ?

Reply:
no, because Laravel have hardcode "email" column name on recovery process

Reply:
From what I see it's a public function that we can override, so it's not hardcoded, it's just defaults I guess. https://github.com/laravel/framework/blob/b9203fca96960ef9cd8860cb4ec99d1279353a8d/src/Illuminate/Auth/Passwords/CanResetPassword.php#L14

We (Backpack) hardcode it here: https://github.com/Laravel-Backpack/CRUD/blob/bad78e72323bedb9e5ac204a651661bf911bf01e/src/app/Library/Auth/PasswordBroker.php#L42

Cheers

Reply:
i do this in PR https://github.com/Laravel-Backpack/CRUD/pull/4623 check thats is working, but @tabacitu want a small fix, this is the alternative way, if you have a clean way to make reset form, email notification and all process work with other column name than "email", let me know.

Cheers.